robot/Cumulus/resources/ObjectMangerPageObject.py

```diff
@@ -20,6 +20,62 @@ def open_fields_and_relationships(self, object_name):
 		search_button = npsp_lex_locators['object_manager']['input'].format("globalQuickfind")
 		self.salesforce.wait_until_loading_is_complete()
 		self.selenium.wait_until_page_contains_element(search_button)
+		if object_name == "Payment":  # Need this extra logic as Payment is not the same as other standard objects
+			self.selenium.get_webelement(search_button).send_keys(object_name)
+			self.selenium.get_webelement(search_button).send_keys(Keys.ENTER)
+			object = npsp_lex_locators['object_manager']['object_result'].format(object_name)
+			leftnavoption = npsp_lex_locators['object_manager']['sidenav_option'].format("Fields & Relationships")
+			self.selenium.wait_until_page_contains_element(object)
+			self.selenium.click_element(object)
+			self.selenium.wait_until_location_contains("Details/view", timeout=90)
+			self.selenium.click_element(leftnavoption)
+			
+	@capture_screenshot_on_error
+	def create_currency_field(self,field_name):
+		""" Creates a custom field of type currency"""
+		currency_locator = npsp_lex_locators['object_manager']['input'].format("dtypeC")
+		next_button = npsp_lex_locators['object_manager']['button'].format("Next")
+		save_button = npsp_lex_locators['object_manager']['button'].format("Save")
+		
+		self.selenium.wait_until_page_contains_element(currency_locator,60)
+		self.selenium.click_element(currency_locator)
+		time.sleep(1)
+		self.selenium.click_element(next_button)
+		self.salesforce.populate_field('Field Label', field_name)
+		self.salesforce.populate_field('Description', "This is a custion field generated during automation")
```

Comment:
please fix the spelling of custom. Also any reason why these two lines are deleted? self.salesforce.populate_field('Length', '16')
self.salesforce.populate_field('Decimal Places', '2')
Currency usually has 2 decimal spaces everywhere so I had put this, so just trying to understand if it was causing any issues

Reply:
While merging my local changes, accidentally my local version got copied. That is not intentional. Updated with the right version